apps/aurora-portal/.env.example

```
@@ -2,7 +2,11 @@ IDENTITY_ENDPOINT="http://localhost:8080/identity/v3/"
 DEFAULT_ENDPOINT_INTERFACE="public"
 PORT="4001"
 CEPH_S3_ENDPOINT="https://rgw.example.com"
-CEPH_REGION="default"
+# Ceph region identifier for AWS Signature V4 signing and LocationConstraint
+# SAP Converged Cloud examples:
+#   - Standard regions: ceph-objectstore-st1 + {region}
+# For other Ceph deployments, use your custom region identifier
+CEPH_REGION="ceph-objectstore-ec-st1-qa-de-1"
 IMAGE_METADATA_EXCLUDED_PROPERTIES="name,tags,visibility,protected,min_disk,min_ram,id,status,size,checksum,created_at,updated_at,created-at,updated-at,disk_format,container_format,file,schema,locations,self,direct_url,owner,virtual_size,kernel_id,ramdisk_id,os_hash_algo,os_hash_value,os-hash-algo,os-hash-value,stores,owner_specified.openstack.md5,owner_specified.openstack.sha256,owner_specified.openstack.object"
 INSECURE_COOKIES=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The INSECURE_COOKIES key should go before the PORT key

(UnorderedKey)